corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -203,8 +208,14 @@ def get_form_data_schema(request, domain, form_unique_id):
         data.append(get_session_schema(form))
         if form.requires_case() or is_usercase_in_use(domain):
             data.append(get_casedb_schema(form))
-    except Exception:
-        logger.exception("schema error")
+    except AppManagerException as e:
+        notify_exception(request, message=e.message)
+        return HttpResponseBadRequest(_(
+            "There is an error in the case management of your application. "
+            "Please fix the error to see case properties in this tree"
```

Comment:
Will it be obvious what that error is and how it should be fixed? I do like this much better than what we had before, but I wonder if it's still likely to wind up reported on fogbugz or field-dev?

Also, what do you think of including the text of the exception message in the error, both here and below?